src/recipes.js

```diff
@@ -4,6 +4,17 @@ const getInstructions = recipe => {
   return recipe.instructions.map(item => item.instruction)
 }
 
+const filterRecipes = (allRecipes, ...tags) => {
+  let filteredRecipes = [];
+  tags.forEach((tag) => {
+    filteredRecipes.push(allRecipes.filter(recipe => {
+      return recipe.tags.includes(tag);
+    }));
+  })
+
+  return [ ...new Set(filteredRecipes.flat())];
```

Comment:
I think we can switch out the `push` functionality in conjunction with the `forEach` to just use a `flatMap`

That way we do not need to push into an array and we do not need `.flat()`

Let me know if further explanation is needed! I can also share a code solution if necessary.